projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -148,7 +154,7 @@ function ( $parent_file ) use ( $menu_slug ) {
 	 * Adds Stats menu.
 	 */
 	public function add_stats_menu() {
-		add_menu_page( __( 'Stats', 'jetpack' ), __( 'Stats', 'jetpack' ), 'edit_posts', 'https://wordpress.com/stats/day/' . $this->domain, null, 'dashicons-chart-bar', 3 );
+		add_menu_page( __( 'Stats', 'jetpack' ), __( 'Stats', 'jetpack' ), 'view_stats', 'https://wordpress.com/stats/day/' . $this->domain, null, 'dashicons-chart-bar', 3 );
```

Comment:
While this works to hide the menu, I can land on the Stats page (and see no stats) if:

- I am an editor on the site.
- I access https://wordpress.com and get redirected to the Stats page, or access `https://wordpress.com/stats/day/site.com` on another site where I'm an admin, and then use the site switcher in the Calypso sidebar to switch to the site where I'm an editor.

This is not part of the scope of this PR, but maybe something that we should fix?

Reply:
Oh right, I think that's a bug in Calypso which defaults to Stats and preserves the current section when switching sites even if the current logged user doesn't have permission.

Likely it's happening right in production as well, with the nav unification disabled.